src/repository/screens/repository.screen.js

```diff
@@ -252,6 +255,7 @@ class Repository extends Component {
                 }
                 loading={isPendingRepository}
                 navigation={navigation}
+                subscribed={isPendingSubscribe ? false : subscribed}
```

Comment:
Just a tiny thing: Could you move this up right below `starred` and follow the same format as `starred`?

i.e.

```
subscribed={
  isPendingRepository || isPendingSubscribe ? false : subscribed
}
```